### DISCUSSION TITLE
Unable to create the time bar in map

### DISCUSSION THREAD

**Discussion by subha-nair** [FlowmapBlue/FlowmapBlue#97] [Q&A]
Unable to create the time bar in map. used a fourth column header time with format 2024-02-04.
![image](https://github.com/user-attachments/assets/2b7ab7c2-d823-41f9-9d54-7502bbcd8dd0)

**Comment by ilyabo** [comment]
There might be an issue with the cell format. Try changing it to "Plain text" for the whole column:

<img width="811" alt="image" src="https://github.com/user-attachments/assets/043f514b-0c2e-417f-963e-586e3737a648">
